javascript/features/player_settings/test/mock_player_settings.js

```diff
@@ -0,0 +1,69 @@
+// Copyright 2020 Las Venturas Playground. All rights reserved.
+// Use of this source code is governed by the MIT license, a copy of which can
+// be found in the LICENSE file.
+
+import SettingList from 'features/player_settings/player_setting_list.js';
+import PlayerSettings from 'features/player_settings/player_settings.js';
+import PlayerSetting from 'features/player_settings/player_setting.js';
+
+import MockPlayerSettingsDatabase from 'features/player_settings/test/mock_player_settings_database.js';
+
+class MockPlayerSettings {
+
+    constructor() {
```

Comment:
If the `PlayerSettingsCommands` would do the writing to the database as well, you could get rid of `MockPlayerSettings` and just always have the same list, regardless of whether tests are being ran. That'd be quite a lot simpler :)